master/maintenance/openstack-upgrade.md

```diff
@@ -75,14 +75,10 @@ This part of the upgrade procedure varies slightly according to which operating
    calico-upgrade complete
    ```
 
-1. Remove any existing `calicoctl` instances and [install the new `calicoctl`](/{{page.version}}/usage/calicoctl/install).
+1. Remove any existing `calicoctl` instances and [install the new `calicoctl`](../getting-started/calicoctl/install).
```

Comment:
This doc seems to have explicit references to v3.1 in it. 

For example, https://github.com/projectcalico/calico/pull/2372/files#diff-b7e3a822caacd3e36fffdff4b3c05921R100

Reply:
Yes, while fixing the broken links, I ended up correcting some errors in older directories.

Reply:
This isn't an older directory though, this is master, so I don't think it should be saying stuff like "Use the following command on the compute nodes to confirm that Felix has upgraded to v3.1.x."

Reply:
That's a good point, let me look into this.